CHANGELOG.md

```diff
@@ -78,6 +78,10 @@ v1.1.0-rc.0
 
 - Fix a bug where custom components would not shadow the stdlib. If you have a module whose name conflicts with an stdlib function
   and if you use this exact function in your config, then you will need to rename your module. (@wildum)
+  
+ - Fixed an issue with `prometheus.scrape` in which targets that move from one
+  cluster instance to another could have a staleness marker inserted and result
+  in a gap in metrics (@thampiotr)
```

Comment:
Just want to double check: is the intent that we're going to release this with 1.1.0 and we need to backport this to the release branch? (It's in the 1.1.0-rc.0 changelog section)